controllers/grid/users/reviewer/form/ThankReviewerForm.php

```diff
@@ -149,7 +149,6 @@ public function execute(...$functionArgs)
         $reviewAssignmentDao = DAORegistry::getDAO('ReviewAssignmentDAO'); /** @var ReviewAssignmentDAO $reviewAssignmentDao */
         $reviewAssignment->setDateAcknowledged(Core::getCurrentDate());
         $reviewAssignment->stampModified();
-        $reviewAssignment->setUnconsidered(ReviewAssignment::REVIEW_ASSIGNMENT_NOT_UNCONSIDERED);
```

Comment:
Removed as don't see why thanking a review needed to update the consideration status .

Reply:
I think the logic is that if an editor is thanking a reviewer they must've taken a look over the review. So I'd suggest following [this same pattern](https://github.com/pkp/pkp-lib/pull/8593/files#diff-f2ed2863fe82393fce36f4a2aa609a7a89260e8cd77e770a36927e439303720aR716) here.

Reply:
Added with slight change .